DTS-STN/Service-Canada-Labs
# Benefits Finder project page

# [Benefits Finder project page](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=208658)

Added Benefits Finder project page and some basic unit and e2e tests.

## Test Instructions

1. See that Benefits Finder card appears on homepage, toggle language
2. Navigate to project page using card, toggle language
